packages/merge-styles/src/Stylesheet.ts

```diff
@@ -211,9 +204,10 @@ export class Stylesheet {
     ) {
       const fabricConfig = global?.FabricConfig || {};
       fabricConfig.mergeStyles = fabricConfig.mergeStyles || {};
-      fabricConfig.mergeStyles.window = fabricConfig.mergeStyles.window ?? win ?? getWindow();
+      fabricConfig.mergeStyles.window =
+        fabricConfig.mergeStyles.window || win || (typeof window !== 'undefined' ? window : undefined);
       fabricConfig.mergeStyles.inShadow = fabricConfig.mergeStyles.inShadow ?? inShadow;
-      fabricConfig.mergeStyles.stylesheetKey = fabricConfig.mergeStyles.stylesheetKey ?? stylesheetKey;
+      fabricConfig.mergeStyles.stylesheetKey = fabricConfig.mergeStyles.stylesheetKey || stylesheetKey;
```

Comment:
Is there a case where `fabricConfig.mergeStyles.stylesheetKey` could be something like `'0'` where this should use `??` instead of `||`?

Reply:
Good call!